src/content/guides/using-persistent-volumes-on-aws-with-efs-csi-driver/index.md

```diff
@@ -0,0 +1,140 @@
+type: page
+weight: 50
+tags: ["tutorial"]
+user_questions:
+  - How do I install the EFS CSI driver?
+  - How do I provision an EFS instance on AWS?
+  - How do I test the EFS CSI storage class?
+  - How do I use EFS CSI Volumes?
+  - What auto-provisioner for AWS do you recommend? EFS? EBS?
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-firecracker
+---
+
+# Using Persistent Volumes on AWS with EFS
+
+If your cluster is running in the cloud on Amazon Web Services (AWS) the most common way to store data is using EBS volumes with the [dynamic provisioner](/guides/using-persistent-volumes-on-aws-with-ebs-csi-driver/). Sometimes EBS is not the optimal solution.
+
+The advantages of using EFS over EBS are:
+
+- EFS data can be accessed from all Availability Zones in the same region while EBS is tied to a single Availability Zone.
+- EFS has the capability to mount the same Persistent Volume to multiple pods at the same time using the ReadWriteMany [access mode](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes).
+- EFS will not hit the [AWS Instance Volume Limit](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/volume_limits.html) as it is a software mount and will avoid the [Impaired EBS](/guides/aws-impaired-volumes/) issue.
+- EFS mount times are better than EBS.
+- EFS provides [encryption in transit](https://aws.amazon.com/blogs/aws/new-encryption-of-data-in-transit-for-amazon-efs/) support using TLS and it's enabled by default.
+
+If you need to use EFS to provision volumes, be advised:
+
+- All Kubernetes Persistent Volumes will be stored in the same EFS instance. You can deploy multiple provisioners per cluster, each having its own storage-class and EFS instance.
+- [EFS throughput](https://docs.aws.amazon.com/efs/latest/ug/performance.html) need to be set up accordingly in order to not have performance issues. We only recommend Provisioned Throughput, and if you need high performance you will need EBS.
+- EFS backups are done with [AWS Backup](https://aws.amazon.com/backup/) and it does not have the snapshot feature of EBS.
+- You cannot limit the amount of data stored in an EFS volume. The requested value in Kubernetes is ignored.
+- EFS mount targets are limited to 1 subnet per Availability Zone. Each NodePool will create a different subnet per AZ, plan accordingly.
+
+## Provision an EFS instance on AWS
+
+**Note:** Currently only static provisioning is supported. This means an AWS EFS file system needs to be created manually on AWS first. After that it can be mounted inside a container as a volume using the driver.
+
+Before installing the provisioner in Kubernetes we will need to create the EFS instance in the same AWS account:
+
+1. Open the [AWS management](https://aws.amazon.com/console/) console in the account your cluster is located.
+2. Select EFS from the services list.
```

Comment:
```suggestion
2. From the `Services` menu, select `EFS`.
```

Putting the terms in the order the user will look for them should make them easier to find. Also backticks make them stand out as literals.